app.js

```diff
@@ -52,7 +53,10 @@ app.locals.franceConnectKitUrl = `${config.FC_URL}${config.FRANCE_CONNECT_KIT_PA
 
 app.get('/', (req, res) => res.render('pages/home'));
 
-app.get('/login', (req, res) => res.status(200).render('pages/login', {}));
+app.get('/login', (req, res) => {
+  const scopesDefined = `${config.MANDATORY_SCOPES} ${config.FC_SCOPES}`.split(' ');
+  return res.status(200).render('pages/login', { scopesDefined, scopesFamilies: GROUPS_FAMILY_SCOPES });
```

Comment:
be more consistant in variable names

instead of: 

scopesFamilies: GROUPS_FAMILY_SCOPES

use:

scopesGroups: SCOPES_GROUPS